ignition-transport2.rb

```diff
@@ -0,0 +1,35 @@
+class IgnitionTransport2 < Formula
+  desc "Transport middleware for robotics"
+  homepage "http://ignitionrobotics.org"
+  url "https://bitbucket.org/ignitionrobotics/ign-transport/get/dfbb1c1.tar.gz"
```

Comment:
Is this URL correct? I see version 1.2.0 in the CMakeLists.txt inside dfbb1c1.tar.gz